dashboard/app/controllers/sms_controller.rb

```diff
@@ -10,8 +10,10 @@ class SmsController < ApplicationController
 
   def send_to_phone
     if params[:level_source] && params[:phone] && (level_source = LevelSource.find(params[:level_source]))
-      send_sms(level_source, params[:phone])
+      send_sms(level_source_url(level_source), params[:phone])
       render status: :ok, nothing: true
+    elsif params[:channel_id] && params[:phone] && %w(artist applab playlab).include?(params[:type])
+      send_sms(eval(params[:type] + '_projects_path') + '#' + params[:channel_id], params[:phone])
```

Comment:
`params[:channel_id]` isn't validated because I don't think it can escape the `#` hash.